src/app/registration/success/facilityConfirmation/diamondHead/diamondHeadConfirmation.component.html

```diff
@@ -0,0 +1,129 @@
+      <!-- Safety information -->
+      <div class="mb-5">
+        <h5 class="mb-4">Safety information</h5>
+        <div class="row mb-3">
+          <div class="col-1">
+            <fa-icon
+              class="text-primary"
+              [icon]="faListCheck"
+              size="2xl"
+            ></fa-icon>
+          </div>
+          <div class="col align-content-center">
+            Be prepared. Create a <a href="https://www.adventuresmart.ca/trip-plan-app/"
+            target="blank"
+            rel="noopener noreferrer
+            "> trip plan </a>and get to know the <a
+              href="https://www.adventuresmart.ca/the-three-ts/"
+              target="_blank"
+              rel="noopener noreferrer"
+            >AdventureSmart's 3Ts</a>.
+          </div>
+        </div>
+        <div class="row mb-3">
+          <div class="col-1">
+            <fa-icon
+              class="text-primary"
+              [icon]="faTriangleExclamation"
+              size="2xl"
+            ></fa-icon>
+          </div>
+          <div class="col align-content-center">
+            Check the <a
+              [attr.href]="parkLink"
+              target="_blank"
+              rel="noopener noreferrer"
+            >park page</a> for advisories.
+          </div>
+        </div>
+        
+        <div class="row mb-3">
+            <div class="col-1">
+              <fa-icon
+                class="text-primary"
+                [icon]="faPawClaws"
+                size="2xl"
+              ></fa-icon>
+            </div>
+            <div class="col align-content-center">
+              Be <a
+                [attr.href]="bearLink"
+                target="_blank"
+                rel="noopener noreferrer"
+              >Bear Aware</a>. Avoid or stow attractants to help avoid encounters.
+            </div>
+          </div>
+
+        <div class="row mb-3">
+            <div class="col-1">
+              <fa-icon
+                class="text-primary"
+                [icon]="faDroplet"
+                size="2xl"
+              ></fa-icon>
+            </div>
+            <div class="col align-content-center">
+              Pack enough water, or be prepared to treat fresh water. There are no signifigant water sources beyond the 5 km mark on the trail untill you reach Elfin Lakes at 11 km. 
```

Comment:
Typo fix: should be "until"

Reply:
resolved in [commit](https://github.com/bcgov/parks-reso-public/pull/439/commits/7489c6211f4b828fb38220544d46c3d372179b3f)